docs/guides/kubernetes-dashboard.md

```diff
@@ -10,363 +10,104 @@ description: >-
 
 # Securing Kubernetes Dashboard
 
-The following guide covers how to secure [Kubernetes Dashboard] using Pomerium. Kubernetes Dashboard is a powerful, web-based UI for managing Kubernetes clusters. Pomerium can act as a **forward-auth provider** _and_ as an independent **identity-aware access proxy** improving and adding single-sign-on to Kubernetes Dashboard's default access control. This guide aims to demonstrate a concrete example of those two methods of access control.
+The following guide covers how to secure [Kubernetes Dashboard] using Pomerium. Kubernetes Dashboard is a powerful, web-based UI for managing Kubernetes clusters. Pomerium can act as an independent **identity-aware access proxy** improving and adding single-sign-on to Kubernetes Dashboard's default access control. This is in contrast to most deployments, which use static tokens for access.
 
 ![fresh kubernetes dashboard install](./img/k8s-fresh-dashboard.png)
 
+
 This tutorial covers:
 
-- Installing [Helm] a package manager for Kubernetes
-- Deploying [NGINX Ingress Controller]
-- Install and configure [Cert-Manager] to issue [LetsEncrypt] certificates
-- Deploying Pomerium
-- Deploying [Kubernetes Dashboard]
-- Secure Kubernetes Dashboard access:
-  - _directly_, using Pomerium's proxy component
-  - _indirectly_, using Pomerium as a [forward-auth] provider
+- Deploying [Kubernetes Dashboard] using [Helm]
+- Secure Kubernetes Dashboard access through Pomerium
```

Comment:
I'm torn on this. I agree that the proposed change would make it more consistent with above, but as a standalone sentence I prefer "Secure" as an adjective versus "Securing" as a verb.

The verb form implies that access exists already and we're modifying it, when actually there's no access until we create an Ingress for it, which will be Secure because it's using our Ingress Controller.

Reply:
Contextually it needs a verb, but maybe...

```suggestion
- Establishing secure Kubernetes Dashboard access through Pomerium
```